tests/pxScene2d/test_pxTimerNative.cpp

```diff
@@ -52,7 +52,8 @@ class pxTimerNativeTest : public testing::Test
         startTime = pxSeconds();
         pxSleepMS(TEST_TIME * 1000);
         endTime = pxSeconds();
-        EXPECT_TRUE((int)(endTime - startTime) == TEST_TIME);
+        bool result = (((int)((endTime - startTime)+0.001) == TEST_TIME)) || (((int)((endTime - startTime)-0.001) == TEST_TIME));
+        EXPECT_TRUE(result == true);
```

Comment:
This change still requires too much accuracy.  Please change to something like

`abs(TEST_TIME - (endTime-startTime)) <= 0.001`